ddtrace/tracer/osinfo_linux.go

```diff
@@ -0,0 +1,51 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-2020 Datadog, Inc.
+
+package tracer
+
+import (
+	"bufio"
+	"os"
+	"strings"
+)
+
+func osName() string {
+	if f, err := os.Open("/etc/os-release"); err == nil {
```

Comment:
Let's have the standard happy path gravitating to the left as usual. 
```suggestion
	if f, err := os.Open("/etc/os-release"); err != nil {
```